.github/ISSUE_TEMPLATE/open_community_working_meeting.md

```diff
@@ -7,6 +7,15 @@ about: Regular Open Community Working Meetings Issue - This template is for thos
 
 Zoom Meeting link: https://postman.zoom.us/j/89562933116?pwd=OWlsQ0RrcDY4S1JQU2d2Q2M0aFFlZz09
 
+**Meeting accesibility**:
+- We would like to record the meeting, for future reference and for those who are not able to attend in person. 
+- We invite you feel comfortable to challenge any language or behaviour that is harmful or not inclusive during this meeting. 
```

Comment:
There's a missing word here. I think it's intended to say "We invite you _to_ feel comfortable". I thought this sentence was a little awkward, so I attempted to suggest an alternate phasing. Not sure if it's better, but sharing anyway.

```suggestion
- We encourage you to challenge any language or behavior that is harmful or not inclusive during this meeting. 
```